src/foundry/client/applications/apps/document-sheet-config.d.mts

```diff
@@ -36,6 +51,51 @@ declare namespace DocumentSheetConfig {
       DocumentSheetV2.Configuration<Document> {}
 
   interface RenderOptions extends HandlebarsApplicationMixin.RenderOptions, DocumentSheetV2.RenderOptions {}
+
+  /** @deprecated Renamed to SheetRegistrationOptions to match core */
```

Comment:
Nit; I like to put `@deprecated` things at the end of a namespace. It makes it easier to delete when we're done.